apps/web/utils/numbers.ts

```diff
@@ -1,11 +1,12 @@
 import * as dn from "dnum";
 import { formatUnits } from "viem";
 
-export const PRECISION_SCALE = BigInt(10 ** 4);
 export const INPUT_MIN_VALUE = 0.000000000001;
 export const MAX_RATIO_CONSTANT = 0.77645;
-export const PERCENTAGE_PRECISION = 10 ** 7;
-export const PERCENTAGE_PRECISION_DECIMALS = 7;
+export const PERCENTAGE_PRECISION = 10 ** 6;
+export const PERCENTAGE_PRECISION_DECIMALS = 6;
```

Comment:
That dont look right. that should 10 ** 4 if you missing 2 in someplace its becuase that need result in mult for 100 in some place.

Reply:
it does not look right i know but neither this 1% = 1e4 = 10.000
So when i am doing math i need a way to represent 1% as 0.01 because 1% = 1/100
And if in the contracts 1% = 10.000 then when i use 0.01 to get the 1% of something i have to multiply the result first by 100 (or add 2 decimals) and then add 4 more decimals (because of the scale)
result: 0.01 * 10 ** 6 = 10000

So i either define the scale with 6 decimals or i add 2 decimals in every percentage calculation in the code.